### ISSUE TITLE
Postman Freeze after 2-3 Operations

### ISSUE THREAD

**Issue by DeepakDani7** [a85/POSTMan-Chrome-Extension#10135]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

Description
Version 6.7.4 
Mac OS: 10.13.6
App Type: Desktop App

Reproduce
Open postman

Sign in
Hit APIs
Freezes intermittently
This is happening from last 10 days . Have to force close it and restart after about 2-3 API request. again its freeze . Please provide solution for this

### Steps To Reproduce

Hit 2-3 APIs
Freezes intermittently
This is happening from last 10 days . Have to force close it and restart after about 2-3 API request. again its freeze . Please provide solution for this

### Environment Information

```markdown
- Operating System: Mac
- Platform Type: Postman Wev
- Postman Version: 6.7.4
```

**Comment by DeepakDani7** [comment]
please do the needful ASAP

**Comment by postman-staff** [comment]
Hi there,

Thanks for reaching out and sorry to see you are having trouble using Postman.

The latest version of the app is v8.8.0 and from the description, you've provided I can see you are on v6.7.4. Could you update the app to the latest version (settings->update) or you can also download the app from our downloads page - https://www.postman.com/downloads/.

Meanwhile, to get you unblocked could navigate the below links to check if Postman on Web works as expected for you? Postman on Web has the same UI & features as that of the desktop app, so feel free to continue using it, if that works as expected.

- Postman on Web for Team users - https://go.postman.co/
- Postman on Web for Single/non-team users - https://web.postman.co/

Let me know if you have any other queries.

via @Swapnil-getpostman | Postman Support

**Comment by postman-staff** [comment]
Hi there, 

Since I did not hear back from you for a while, so just letting you know that I'm going ahead and closing this conversation for now, but I’m still here so feel free to respond back to me on this thread if you have any further queries :)

Alternatively, feel free to submit a new ticket on any matter and at any time using this link - https://www.postman.com/support/  and will be happy to help.

via @Swapnil-getpostman | Postman Support

**Comment by DannyDainton** [comment]
Closing this due to inactivity
